plugins/storage/volume/storpool/src/main/java/org/apache/cloudstack/storage/motion/StorPoolDataMotionStrategy.java

```diff
@@ -219,24 +219,23 @@ public void copyAsync(DataObject srcData, DataObject destData, Host destHost,
                 } else {
                     answer = (CopyCmdAnswer) ep2.sendMessage(backupSnapshot);
                     if (answer != null && answer.getResult()) {
-                        SpApiResponse resSnapshot = StorPoolUtil.volumeFreeze(volumeName, conn);
+                        SpApiResponse resSnapshot = StorPoolUtil.volumeSnapshot(volumeName, template.getUuid(), null, "template", null, conn);
                         if (resSnapshot.getError() != null) {
-                            logger.debug("Could not snapshot volume [id: {}, name: {}]", snapshot.getId(), snapshot.getName());
-                            StorPoolUtil.spLog("Volume freeze failed with error=%s", resSnapshot.getError().getDescr());
+                            logger.debug(String.format("Could not snapshot volume with ID=%s", snapshot.getId()));
```

Comment:
why revert to the old style logging? 
```suggestion
                            logger.debug("Could not snapshot volume with ID={}", snapshot.getId());
```
should work

Reply:
Thanks, @DaanHoogland, I haven't noticed the change while rebasing. I will update my changes

Reply:
changed in the latest commit